src/components/ButtonPanel/ButtonPanel.css

```diff
@@ -0,0 +1,4 @@
+.ButtonPanel {
```

Comment:
Why don't we just use `className` in our `Button` to indicate button color instead of using inline styling there?